lib/puppet/provider/zabbix.rb

```diff
@@ -1,5 +1,5 @@
-# encoding: utf-8
 # zabbix provider type for puppet
+# rubocop:disable Style/ClassAndModuleChildren
```

Comment:
That would force us to use:
```ruby
class Puppet
  class Provider
    class Zabbix
    end
  end
end
```

But it would overwrite the Puppet/Provider classes.